specification/resources/resource-manager/Microsoft.Authorization/stable/2023-04-01/policySetDefinitionVersions.json

```diff
@@ -0,0 +1,915 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PolicyClient",
+    "version": "2023-04-01",
+    "description": "To manage and control access to your resources, you can define customized policies and assign them at a scope."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Authorization/listPolicySetDefinitionVersions": {
+      "post": {
+        "tags": [
+          "PolicySetDefinitionVersions"
+        ],
+        "operationId": "PolicySetDefinitionVersions_ListAllBuiltins",
+        "summary": "Lists all built-in policy set definition versions.",
+        "description": "This operation lists all the built-in policy set definition versions for all built-in policy set definitions.",
+        "x-ms-examples": {
+          "List all built-in policy definition versions": {
+            "$ref": "./examples/listAllBuiltInPolicySetDefinitionVersions.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of policy set definition versions.",
+            "schema": {
+              "$ref": "#/definitions/PolicySetDefinitionVersionListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
```

Comment:
>x-ms-pageable [](http://example.com/codeflow?start=9&length=13)

The x-ms-pageable extension doesn't actually do anything on POST. SDK paging support only works on GET, so you need to write your own paging code for POST.

Reply:
understood...  Does this need to be removed?

Reply:
removed